lib/credit_card_form.dart

```diff
@@ -180,7 +180,7 @@ class _CreditCardFormState extends State<CreditCardForm> {
   late CreditCardModel creditCardModel;
 
   final MaskedTextController _cardNumberController =
-      MaskedTextController(mask: '0000 0000 0000 0000');
+      MaskedTextController(mask: '0000 0000 0000 0000 000');
```

Comment:
These changes should be based on the brand, as most of the brands support only 16 digits.